### DISCUSSION TITLE
[Suggestion] Allow customers to create a public "donation" link to add credits

### DISCUSSION THREAD

**Discussion by NoPantsMcDance** [Paymenter/Paymenter#291] [Ideas]
It would be neat if a customer could create a public "donation" link for people to add credits to their account. 

This would be useful for game servers where their player base would know that their donation goes to the cost of the server.

**Comment by finnjosten** [comment]
Or different idea to this is just to add a direct pay, a user can chose how much they pay and you can set a min and max a users can/has to pay with that link
